infra/build_specified_commit.py

```diff
@@ -22,75 +22,81 @@
 import subprocess
 
 import helper
-import repo_manager
 
 
 class DockerExecutionError(Exception):
   """An error that occurs when running a docker command."""
 
 
-def build_fuzzer_from_commit(project_name,
-                             commit,
-                             local_store_path,
-                             engine='libfuzzer',
-                             sanitizer='address',
-                             architecture='x86_64',
-                             old_repo_manager=None):
+def build_fuzzers_from_commit(project_name,
+                              commit,
+                              build_repo_manager,
+                              engine='libfuzzer',
+                              sanitizer='address',
+                              architecture='x86_64'):
   """Builds a OSS-Fuzz fuzzer at a  specific commit SHA.
 
   Args:
     project_name: The OSS-Fuzz project name
     commit: The commit SHA to build the fuzzers at
-    local_store_path: The full file path of a place where a temp git repo is stored
+    build_repo_manager: The OSS-Fuzz project's repo manager to be built at
     engine: The fuzzing engine to be used
     sanitizer: The fuzzing sanitizer to be used
     architecture: The system architiecture to be used for fuzzing
 
   Returns:
     0 on successful build 1 on failure
   """
-  if not old_repo_manager:
-    inferred_url, repo_name = detect_main_repo_from_docker(project_name, commit)
-    old_repo_manager = repo_manager.RepoManager(
-        inferred_url, local_store_path, repo_name=repo_name)
-  old_repo_manager.checkout_commit(commit)
+  build_repo_manager.checkout_commit(commit)
   return helper.build_fuzzers_impl(
       project_name=project_name,
       clean=True,
       engine=engine,
       sanitizer=sanitizer,
       architecture=architecture,
       env_to_add=None,
-      source_path=old_repo_manager.repo_dir,
-      mount_location=os.path.join('/src', old_repo_manager.repo_name))
+      source_path=build_repo_manager.repo_dir,
+      mount_location=os.path.join('/src', build_repo_manager.repo_name))
```

Comment:
ehh not great that this `/src` is hardcoded since it should really be gotten from the env var `$SRC` in the image.